src/python/pants/backend/scala/subsystems/scalac.py

```diff
@@ -26,17 +26,36 @@ class Scalac(Subsystem):
     plugins_global = StrListOption(
         "--plugins-global",
         help=(
-            "A list of addresses of `scalac_plugin` targets which should be used for "
+            "DEPRECATED: A list of addresses of `scalac_plugin` targets which should be used for "
```

Comment:
Maybe set removal version and hint?

Reply:
you do that by using the method `.deprecated()`